src/planning.py

```diff
@@ -219,9 +220,9 @@ def _skeleton_generator(
         _, _, node = hq.heappop(queue)
         # Good debug point #1: print out the skeleton here to see what
         # the high-level search is doing. You can accomplish this via:
```

Comment:
we should revert this part, it'll just dominate the printouts when running with debug level. let's leave it commented out